scripts/python_env.sh

```diff
@@ -3,12 +3,20 @@
 ####  Usage ####
 # source python_env.sh [debug | release]
 
-# Check build type
+# Check build type parameter
 if [[ -z $1 ]]; then
         echo "build type must be specified. "
         echo "Usage: $0 [debug | release ]"
-        return
+        return 1
 fi
+BUILD_TYPE=$1
+BUILD_TYPE=`echo $BUILD_TYPE | tr "[A-Z]" "[a-z]"` # convert to lower case
+if [ $BUILD_TYPE != "debug" ] && [ $BUILD_TYPE != "release" ]; then
```

Comment:
I think you'll want quotes on the left hand side of each string comparison as well, like `"$BUILD_TYPE" != "debug"`, in case the $BUILD_TYPE variable is set to empty string (which will cause a syntax error in the absence of quotes).

Reply:
Fixed. I didn't know that would be a syntax error. Thanks.